posts/inside-rust/2022-05-19-governance-update.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Governance Update"
+author: Ryan Levick and Mara Bos
+---
+
+Last month, the core team, all the leads of top-level teams, the moderators, and the project representatives on the Rust Foundation board jointly sent out an update to all Rust project members on investigations happening into improvements to the governance of the Rust project. 
+
+As is detailed in the update, this report follows extensive conversations with many inside of the project and will continue as we encourage others in the project to reach out should they have feedback or want to get involved. 
+
+Since proper governance of the Rust project impacts all users of Rust, we thought it appropriate to also make this summary public. While typos and grammatical fixes have been made since the original post, the text is otherwise unaltered. [^1]
+
+As is noted in the summary, the next steps are to take the findings we have so far and begin crafting proposals for how the Rust project will be governed in the future. This will eventually lead to RFCs or similar documents with concrete proposals.
+
+[^1]: See [rust-lang/blog.rust-lang.org#974](https://github.com/rust-lang/blog.rust-lang.org/pull/974) for the full history of changes to the original email.
+
+> From: Ryan Levick\
+> To: All members of the Rust project\
+> Date: Mon, 11 Apr 2021 18:27:00 UTC\
```

Comment:
Shouldn't the date be 2022?

Reply:
Yes.

Reply:
https://github.com/rust-lang/blog.rust-lang.org/pull/975